tests/test_config.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=all
+import pathlib
+import pytest
+import os
+
+from uwtools.configure import Configure
+from uwtools.template import Template,TemplateConstants
+
+uwtools_file_base = os.path.join(os.path.dirname(__file__))
+
+def test_configuation_parse_env():
+
+    os.environ['TEST'] = 'TEST_TRUE'
+    config = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/experiment.yaml")))
+
+    expected = os.environ.get('TEST')
+    actual = config.test_env
+    assert actual == expected
+
+
+def test_configuation_update():
+
+    config = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/experiment.yaml")))
+    config.include(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/gfs.yaml")))
+
+    expected =  "/home/myexpid/{{current_cycle}}"
+    actual = config.datapath
+
+    assert actual == expected
+
+
+def test_configuation_update_object():
+
+    config = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/experiment.yaml")))
+    config2 = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/gfs.yaml")))
+    config.include(data=config2)
+
+    expected =  "/home/myexpid/{{current_cycle}}"
+    actual = config.datapath
+
+    assert actual == expected
+
+
+def test_configuration_inplace_update():
+
+    config = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/gfs.yaml")))
+
+    expected =  "testpassed"
+    actual = config.testupdate
+
+    assert actual == expected
+
+def test_configuration_realtime_update():
+
+    config = Configure(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/experiment.yaml")))
+    config.include(pathlib.Path(os.path.join(uwtools_file_base,"fixtures/gfs.yaml")))
+    config = Template.substitute_structure( config, TemplateConstants.DOUBLE_CURLY_BRACES, config.get)
```

Comment:
Tests often serve as a use example. Is this how the user would need to load this set of configuration files?

It is also not clear what parts of the yaml parsing this test is covering. A doc block would _definitely_ help here.

Reply:
OK, I'll work on these a bit more in my branch

Reply:
As a whole the config object in this PR is a place holder for the expanding the capabilities in YAMAL to include the stringification and includes, I should have made that clearer in the self documentation.

Reply:
Updated explanations in code remarks